src/test/java/core/basesyntax/PurchaseOperationTest.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import core.basesyntax.service.operations.PurchaseOperation;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseOperationTest {
+    private static PurchaseOperation purchaseOperation;
```

Comment:
```suggestion
    private static Operation purchaseOperation;
```
let's use interface for the field reference type